### DISCUSSION TITLE
Haxeui Errors

### DISCUSSION THREAD

**Discussion by RuiCuco** [Kode/Kha#1457] [General]
Hi. I just installed the latest versions of everything and I get the following errors:

C:\Developer\HaxeToolkit\haxe\lib\haxeui-kha\1,6,0/haxe/ui/backend/kha/SDFPainter.hx:846: characters 43-64 : Class<kha.Shaders> has no field painter_sdf_rect_frag
C:\Developer\HaxeToolkit\haxe\lib\haxeui-kha\1,6,0/haxe/ui/backend/kha/SDFPainter.hx:847: characters 41-62 : Class<kha.Shaders> has no field painter_sdf_rect_vert
C:\Developer\HaxeToolkit\haxe\lib\haxeui-kha\1,6,0/haxe/ui/backend/kha/SDFPainter.hx:869: characters 43-66 : Class<kha.Shaders> has no field painter_sdf_circle_frag
C:\Developer\HaxeToolkit\haxe\lib\haxeui-kha\1,6,0/haxe/ui/backend/kha/SDFPainter.hx:870: characters 41-64 : Class<kha.Shaders> has no field painter_sdf_circle_vert
C:\Developer\HaxeToolkit\haxe\lib\haxeui-kha\1,6,0/haxe/ui/backend/kha/SDFPainter.hx:891: characters 43-64 : Class<kha.Shaders> has no field painter_sdf_line_frag
C:\Developer\HaxeToolkit\haxe\lib\haxeui-kha\1,6,0/haxe/ui/backend/kha/SDFPainter.hx:892: characters 41-62 : Class<kha.Shaders> has no field painter_sdf_line_vert
C:\Developer\Kha\Tools\windows_x64\std/js/_std/Std.hx:87: characters 36-42 : Object is not a value

(Notice in particular the last error.) This was working fine before the updates. Can anyone help?

Also, VS Code doesn't list now all the haxe dependencies, especially the haxeui-core and the haxeui-kha ones. Can anyone tell me why?

**Comment by RobDangerous** [comment]
Sorry, I don't use HaxeUI myself. But I'm sure Ian (aka Mr HaxeUI) will help if you ask him.
